xtask/src/commands/test.rs

```diff
@@ -2,7 +2,17 @@ use anyhow::Result;
 use structopt::StructOpt;
 use xtask::*;
 
-const TEST_DEFAULT_ARGS: &[&str] = &["test", "--all", "--locked"];
+const TEST_DEFAULT_ARGS: &[&str] = &[
+    "test",
+    "--manifest-path",
+    "apollo-router/Cargo.toml",
+    "--locked",
+    "--jobs",
+    "4",
+    "--",
+    "--test-threads",
+    "6",
+];
```

Comment:
CI is not affected as it uses `cargo test` rather than `cargo xtask test`. Is it intended to make these changes for contributors testing locally?

Reply:
CI now uses `xtask test`. I'm going to be following up with a PR that will give us thread dumps if tests hang, and this will be unified for local dev and CI.

Reply:
https://github.com/apollographql/router/pull/2684/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R352